NOnion/Directory/TorDirectory.fs

```diff
@@ -237,15 +237,14 @@ type TorDirectory =
                 self.NetworkStatus <- NetworkStatusDocument.Parse response
         }
 
-    static member Bootstrap
-        (nodeEndPoint: IPEndPoint)
+    static member BootstrapWithGuard
```

Comment:
Each public API function needs an async version.

Reply:
It's already async

Reply:
I'm guessing he means a member with the same name but ended with Async, consumable by C#, which just calls BootstrapWithGuard but runs the async job and returns the task

Reply:
done